packages/flutter/lib/src/material/calendar_date_picker.dart

```diff
@@ -1013,6 +1013,8 @@ class _DayPickerState extends State<_DayPicker> {
               // for the day of month. To do that we prepend day of month to the
               // formatted full date.
               label: '${localizations.formatDecimal(day)}, ${localizations.formatFullDate(dayToBuild)}$semanticLabelSuffix',
+              // We set button to true to make the date selectable.
```

Comment:
avoid using `we` or `you` in comments. 
```suggestion
              // Set button to true to make the date selectable.
```

Reply:
Fixed:) Thanks!